### PR TITLE
Initial version

### LINKED ISSUE #1: Create command line tool that validates XML against a Schema
Related to https://github.com/CSCfi/metadata-submitter we need a tool that validate an XML from CLI.

We can use argparse or click to create the tool.

### PR DESCRIPTION
**Pull request by csc-jm**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Metadata submitter tools
+
+## XML Validation CLI
+
+Command line tool for validating a given XML file against a specific XSD Schema.
+
+### Installing
+
+```
```
Comment (blankdots) [inline README.md:9]: let us not have this venv installation. we can just require python3.6+

**Comment by blankdots** [comment]
`setup.py` needs to be in root directory

File: xml-validator/setup.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    name='xml-validator',
+    version='0.1',
+    py_modules=['validator'],
+    install_requires=[
+        'Click',
+        'xmlschema',
+    ],
+    entry_points='''
+        [console_scripts]
+        xml-validator=validator:cli
```
Comment (blankdots) [inline xml-validator/setup.py:13]:
```suggestion
        xml-validator=xml-validator:validator:cli
```

move `setup.py` to root directory and the entry point will look like above, however might be a good idea to rename the entry point to not be the same name as the folder of the python package

File: xml-validator/validator.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    except xmlschema.validators.exceptions.XMLSchemaValidationError as err:
+        click.secho(f"{click.format_filename(xml_file)} is invalid.\n", fg='red')
+        if verbose:
+            click.secho("Error:", bold=True)
+            click.echo(err)
+
+    except xml.etree.ElementTree.ParseError as err:
+        click.echo("Faulty XML or XSD file was given.\n")
+        if verbose:
+            click.echo(f"Error: {err}")
```
Comment (blankdots) [inline xml-validator/validator.py:26]: this would be good to add: https://docs.python.org/3.7/library/__main__.html that calls `cli`

**Comment by csc-jm** [comment]
fixed the requested fixed and added requirements.txt

File: validator/__main__.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        click.secho(f"{click.format_filename(xml_file)} is invalid.\n",
+                    fg='red')
+        if verbose:
+            click.secho("Error:", bold=True)
+            click.echo(err)
+
+    except xml.etree.ElementTree.ParseError as err:
+        click.echo("Faulty XML or XSD file was given.\n")
+        if verbose:
+            click.echo(f"Error: {err}")
```
Comment (blankdots) [inline validator/__main__.py:27]:
```suggestion
            click.echo(f"Error: {err}")
            
if __name__ == "__main__":
    cli()
```

still missing this in order to run it as a module `python -m validator` from root directory

Comment (csc-jm) [reply validator/__main__.py:27]: Ah, I see

**Pull request merged**